sensor_main.py

```diff
@@ -0,0 +1,141 @@
+#!/bin/python3
```

Comment:
Can we `#!/usr/bin/env python` to align with bash files from other repos?
If it's the case in other files, please align them as well.